packages/storage-ui/src/Components/Modules/FileSystemItem/FileSystemGridItem.tsx

```diff
@@ -234,7 +234,7 @@ const FileSystemGridItem = React.forwardRef(
           >
             {icon}
           </div>
-          {editing === cid && desktop ? (
+          {editing?.cid === cid && editing.name === name && desktop ? (
```

Comment:
pretty sure I broke this in the past, because I didn't realize the name made sense for folders for MFS, since they don't have a unique cid
```suggestion
          // checking the name is useful for MFS folders since empty folders all have the same cid
          {editing?.cid === cid && editing.name === name && desktop ? (
```